fzf-marks.plugin.bash

```diff
@@ -165,11 +159,148 @@ function dmark {
     setup_completion
 }
 
+if ((BASH_VERSINFO[0] >= 4)); then
+    # Widget for Bash 4.0+
+
+    # bashbug https://lists.gnu.org/archive/html/bug-bash/2018-04/msg00040.html
+    function _fzm-widget-has_readline_point_bug {
```

Comment:
Also, do you have any suggestions for the name of the private `fzm-marks` functions? Currently, I use `_fzm` as the private prefix and connect names with hyphens. However, the conventional naming of shell functions only uses the underscore in addition to the alphanumeric letters but not the hyphen.

Reply:
I think `_fzm` is very good!

Reply:
Thank you! I also changed the names of the other private functions (the functions of internal use of `fzf-marks`). I think we should add the prefix to all the internal functions. For example, `setup_complete` could conflict with the function name from the other plugins or user settings of a similar purpose.

Reply:
Now that you point this out, I completely agree! Thank you!